packages/botonic-react/src/constants.js

```diff
@@ -125,8 +120,16 @@ export const WEBCHAT = {
     // TriggerButton
     customTrigger: 'triggerButton.custom',
     triggerButtonImage: 'triggerButton.image',
-    triggerButtonNotificationsEnabled: 'triggerButton.notificationsEnabled',
+    // triggerButtonNotificationsEnabled: 'triggerButton.notificationsEnabled',
     triggerButtonStyle: 'triggerButton.style',
+    // Notifications
+    notificationsEnabled: 'notifications.enable',
+    notificationsBannerCustom: 'notifications.banner.custom',
+    notificationsBannerText: 'notifications.banner.text',
+    notificationsTriggerButtonEnabled: 'notifications.triggerButton.enable',
+    // Scroll Button
+    scrollButtonEnabled: 'scrollButton.enable',
+    scrollButtonCustom: 'scrollButton.custom',
```

Comment:
Yes, I add it and use it to activate the banner. I would like to discuss with the Services team how to activate all these components, I think it will be too custom and may be confusing.

Reply:
Applied